components/hab/src/command/studio/enter.rs

```diff
@@ -132,10 +133,29 @@ mod inner {
         return false;
     }
 
-    fn rerun_with_sudo_if_needed(ui: &mut UI) -> Result<()> {
-        // If I have root permissions, early return, we are done.
+    fn has_docker_group() -> bool {
+        let groupname = "docker".to_string();
+        let current_user = group::get_current_username().unwrap();
+        let docker_members = group::get_members_by_groupname(&groupname);
+        let mut _ret = false;
+        for user in docker_members {
+            if user == current_user {
+                _ret = true;
+                break;
+            }
+        }
+        _ret
+    }
+
+    fn rerun_with_sudo_if_needed(ui: &mut UI, args: &Vec<OsString>) -> Result<()> {
```

Comment:
While `&Vec<OsString>` is correct, you hardly ever see it written like that.  Usually you see it written as a slice, e.g. `&[OsString]`.